docs/how-to-guides/feathr-configuration-and-env.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: default
+title: Configuration, environment variables, and store secrets in a secure way
```

Comment:
Suggest add a link to this doc in https://github.com/linkedin/feathr/blob/main/feathr_project/feathrcli/data/feathr_user_workspace/feathr_config.yaml

Reply:
good point, will add a link to that once we merge this to main.